src/Avalonia.Controls/Window.cs

```diff
@@ -225,8 +225,14 @@ public void Close(object dialogResult)
         /// </summary>
         public override void Hide()
         {
+            if (!IsVisible)
+            {
+                return;
+            }
+
             using (BeginAutoSizing())
             {
+                Renderer?.Stop();
```

Comment:
Do you mean that we _also_ need to react to `WindowState` changes, or that some other form of window state should _replace_ `Hide`?